core-services/secret-mirroring/_mapping.yaml

```diff
@@ -138,6 +138,12 @@ secrets:
   to:
     namespace: test-credentials
     name: test-harness-secrets
+- from:
+    namespace: crw-ci-secrets
+    name: ocm-aws-access
+  to:
+    namespace: ci
```

Comment:
Depending how exaclty you want to use the secret, it's pretty likely that you need to have the secret in `test-credentials` namespace, not in the `ci` one.

Reply:
/hold
Holding to figure out where this secret needs to go

Reply:
@petr-muller we plan to just map them as env variables using `fromSecret` inside `ci-operator/jobs/openshift/osde2e/openshift-osde2e-main-periodics.yaml`

Reply:
ok, so that means handcrafted jobs and directly mounting that secret in a Prowjob. So in that case `ci` is correct.

Reply:
could you also remove the `hold` label as well in that case, please?